legend-engine-xt-graphQL-pure/src/main/resources/core_external_query_graphql/deprecated/tests/generationTests.pure

```diff
@@ -145,12 +148,10 @@ function <<test.Test>> meta::external::query::graphQL::generation::tests::testGe
     'meta/external/query/graphQL/generation/tests/model/Person.graphql',
     'meta/external/query/graphQL/generation/tests/model/Query.graphql',
     'meta/external/query/graphQL/generation/tests/model/Query2.graphql',
-    'meta/external/query/graphQL/generation/tests/model/IncType.graphql',
-    'scalars.graphql'
+    'meta/external/query/graphQL/generation/tests/model/IncType.graphql'
```

Comment:
I think we did a separate scalars.graphql to avoid duplication .. like this https://github.com/apollographql/apollo-server/issues/2767 @abhishoya-gs might have more details. Maybe spring grapqhl had an issue with duplicates ?

Reply:
Yes, spring does throw an error on redefining. And it seems we might run into issues in other graphql engines such has apollo.

Reply:
Ok will undo this piece then.